tests/core/data/test_data_model.py

```diff
@@ -14,80 +14,16 @@
 
 
 class TestDataModel:
-    def test_deprecated_properties(self):
-        model = _DataNodeModel.from_dict(
-            {
-                "id": "id",
-                "config_id": "config_id",
-                "scope": repr(Scope.SCENARIO),
-                "storage_type": "pickle",
-                "name": "name",
-                "parent_id": "owner_id",
-                "parent_ids": ["parent_id"],
-                "last_edition_date": "2020-01-01T00:00:00",
-                "read_fct_name": "read_fct_name",
-                "read_fct_module": "read_fct_module",
-                "write_fct_name": "write_fct_name",
-                "write_fct_module": "write_fct_module",
-                "job_ids": [],
-                "edits": [],
-                "version": "latest",
-                "cacheable": False,
-                "validity_days": 1,
-                "validity_seconds": 1,
-                "edition_in_progress": False,
-                "data_node_properties": {},
-            }
-        )
-        assert model.edit_in_progress is False
-        assert model.last_edit_date == "2020-01-01T00:00:00"
-        assert model.owner_id == "owner_id"
-        assert model.data_node_properties.get("cacheable") is None
-
-    def test_override_deprecated_properties(self):
```

Comment:
I would revert this and rename the test so we know we are testing last_edition_date.